client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/standard-agency-hosting/wpcom-plan-selector/index.tsx

```diff
@@ -25,6 +26,8 @@ type PlanDetailsProps = {
 	setQuantity: ( quantity: number ) => void;
 };
 
+const MAX_SLIDER_POINTS = 10;
```

Comment:
What do you think about renaming this to `MAX_PLANS_FOR_SLIDER` or something else?

Reply:
Thanks for the review, @vitozev!

Makes sense. Updated it here: https://github.com/Automattic/wp-calypso/pull/93402/commits/77b46ca02ebd077eef9dfa8b6ac72587889f7a2f